src/pdb.rs

```diff
@@ -495,6 +495,17 @@ impl DeviceSQLString {
         )
     }
 
+    fn parse_isrc_string(input: &[u8]) -> IResult<&[u8], DeviceSQLString> {
+        let (input, _) = nom::bytes::complete::tag(b"\x90")(input)?;
+        let (input, length) = nom::number::complete::le_u16(input)?;
+        let (input, _) = nom::bytes::complete::tag(b"\x00\x03")(input)?;
+        let (input, data) = nom::bytes::complete::take(length - 6)(input)?;
+        std::str::from_utf8(data).map_or_else(
+            |_| Err(nom_input_error_with_kind(input, ErrorKind::Char)),
+            |text| Ok((input, Self::ShortASCII(text.to_owned()))),
```

Comment:
I think we should add a new string type for this, otherwise we cannot serialize it properly when we add support for that.